src/App/Template/ErrorTemplate.php

```diff
@@ -13,6 +13,23 @@ class ErrorTemplate extends AbstractTemplate
 {
     use HandlerAwareTrait;
 
+    /**
+     * Gets the data keys on this entity.
+     * @see
+     *
+     * @return array
+     */
+    public function keys($keys = null)
+    {
+        return array_merge(parent::keys($keys), [
+            'debug',
+            'errorCode',
+            'errorMessage',
+            'errorTitle',
+            'htmlErrorDetails'
```

Comment:
Add trailing comma to multi-line array declarations for easier diffing.

```suggestion
            'debug',
            'errorCode',
            'errorMessage',
            'errorTitle',
            'htmlErrorDetails',
```